### PR TITLE
added students solution to review

### PR DESCRIPTION
**Pull request by spartiv**

### CODE REVIEW

File: src/main/java/core/basesyntax/Ball.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Ball.java:5]: fields must be private

File: src/main/java/core/basesyntax/Ball.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+        return this.ballNumber;
+    }
+
+    public void setBallNumber(int ballNumber) {
+        this.ballNumber = ballNumber;
+    }
+
+    public String toString() {
+        String ball = "ball";
+        return "Ball with number: " + ballNumber + " and color: " + ballColor;
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Ball.java:33]: code style's good practice is to place 
"+"  on the new line

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 3]
```diff
@@ -1,7 +1,12 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+
+
```
Comment (spartiv) [inline src/main/java/core/basesyntax/ColorSupplier.java:7]: redundant empty lines

File: src/main/java/core/basesyntax/Colors.java [hunk 4]
```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
+    white,
+    red,
+    blue
+}
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Colors.java:8]: its considered a good practice 
to have all Enums in capitals

Comment (spartiv) [reply src/main/java/core/basesyntax/Colors.java:8]: no empty line in the end of file

File: src/main/java/core/basesyntax/ColorSupplier.java [hunk 5]
```diff
@@ -1,7 +1,12 @@
...
+    public static String getRandomColor() {
+        int index = new Random().nextInt(Colors.values().length);
+        return Colors.values()[index].toString();
     }
 }
```
Comment (spartiv) [inline src/main/java/core/basesyntax/ColorSupplier.java:12]: no empty line in the end of file

File: src/main/java/core/basesyntax/Lottery.java [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+
+        Ball ball = new Ball();
+
+        ball.setBallColor(ColorSupplier.getRandomColor());
+
+        ball.setBallNumber(new Random().nextInt(100));
+
+        return ball;
+    }
+}
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Lottery.java:16]: and again no empty line in the end of file

File: src/main/java/core/basesyntax/Ball.java [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+    public int getBallNumber() {
+        return this.ballNumber;
+    }
+
+    public void setBallNumber(int ballNumber) {
+        this.ballNumber = ballNumber;
+    }
+
+    public String toString() {
+        String ball = "ball";
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Ball.java:32]: what is this local variable for 
its not used anywhere, please remove

File: src/main/java/core/basesyntax/Ball.java [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
+
+    public Ball(String color, int number) {
+        this.ballColor = color;
+        this.ballNumber = number;
+    }
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Ball.java:13]: it doesn't seem that this constructor is used, and therefore its redundant.

File: src/main/java/core/basesyntax/Ball.java [hunk 9]
```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+public class Ball {
+    String ballColor;
+    int ballNumber;
+
+    public Ball() {
+    }
```
Comment (antonignatenko) [inline src/main/java/core/basesyntax/Ball.java:8]: this const is available by default.No need to announce it.

Comment (spartiv) [inline src/main/java/core/basesyntax/Ball.java:8]: this constructor is redundant

File: src/main/java/core/basesyntax/Application.java [hunk 10]
```diff
@@ -2,6 +2,11 @@
 
 public class Application {
     public static void main(String[] args) {
-        // create three balls using class Lottery and print information about them in console
+        Lottery ball1 = new Lottery();
```
Comment (spartiv) [inline src/main/java/core/basesyntax/Application.java:5]: id call variables a bit more informative
like 
lotteryBall1

so it's clear what type of class it refers to

**Pull request closed without merging**